src/plugin.ts

```diff
@@ -68,58 +68,15 @@ export const updateSnapshot = async (options: UpdateSnapshotOptions): Promise<Vi
   fs.copyFileSync(options.screenshotAbsolutePath, destFile)
   const fileBuffer = fs.readFileSync(destFile)
   logger.info(`Updated base snapshot '${options.screenshotName}' at ${destFile}`)
-  // @ts-expect-error // TODO remove this line after https://github.com/cypress-io/cypress/issues/29048 is fixed
-  moveActualSnapshotIfNeeded(options.screenshotAbsolutePath, options.spec.relativeToCommonRoot)
   return { images: { actual: fileBuffer.toString('base64') }, baseGenerated: true }
 }
 
-/**
- * @description  For some reason Cypress does save the screenshots in the wrong folder when running in headless mode
- * It will use spec file to build an extra folder in the path
- * This is a workaround to remove the extra folder from the path
- * ie:
- * [CORRECT]   cypress open => .../snapshots/actual/cypress/e2e/alt-sub/foo/deep.cy.ts/inside_context.png
- * [INCORRECT] cypress run =>  .../snapshots/actual/deep.cy.ts/cypress/e2e/alt-sub/foo/deep.cy.ts/inside_context.png
- *
- * looking at Cypress.Spec we can see that the relativeToCommonRoot is set only when running in headless mode
- * and that is the only time we need to remove the extra folder from the path
- *
- * ref: https://github.com/cypress-io/cypress/issues/29057
- *
- * @param actualScreenshot
- * @param relativeToCommonRoot
- * @returns boolean
- */
-const moveActualSnapshotIfNeeded = (actualScreenshot: string, relativeToCommonRoot?: string): boolean => {
```

Comment:
@area73 hey, got a question here, was there any reason why there was a need in the first place to move actual screenshots anywhere other than where Cypress itself puts it?
In my opinion, this does not influence the plugin, as we have access to the absolute path of the screenshot to use for comparison or base generation. It could be a historical reason, before we switched to absolute path, but I'm not sure really
Could I be missing any other purpose? 🤔